nextcord/mixins.py

```diff
@@ -47,3 +57,45 @@ class Hashable(EqualityComparable):
 
     def __hash__(self) -> int:
         return self.id >> 22
+
+
+class PinsMixin:
+    __slots__ = ()
+    _state: Any
+
+    if TYPE_CHECKING:
+        _state: ConnectionState
+
+        @overload
+        async def pins(self) -> List[Message]:
+            ...
+
+    async def _get_channel(self):
+        raise NotImplementedError
+
+    async def pins(self) -> List[Any]:
```

Comment:
Any? why

Reply:
I'm not a typing wizard, and importing Message from the messages module causes a circular import
If you know how to make this better let me know

Reply:
by simply using `from __future__ import annotations`

Reply:
Yeah I realized that myself